web/basic/footer.html

```diff
@@ -105,14 +104,20 @@ <h5>Ban Player:</h5>
                                     </label>
                                     <div class="col-sm-9">
                                         <select class="form-control" id="modPlayerBan-durationSelect">
-                                            <option value="2 hours">2 hours</option>
-                                            <option value="8 hours">8 hours</option>
-                                            <option value="1 day">1 day</option>
-                                            <option value="2 days" selected>2 days</option>
-                                            <option value="1 week">1 week</option>
-                                            <option value="2 weeks">2 weeks</option>
-                                            <option value="permanent" class="font-weight-bold text-danger">permanent</option>
-                                            <option value="custom" class="font-weight-bold">custom</option>
+                                            <option value="1 hours">1 hour</option>
+											<option value="2 hours">2 hours</option>
+											<option value="4 hours">4 hours</option>
+											<option value="8 hours">8 hours</option>
+											<option value="12 hours">12 hours</option>
+											<option value="1 day">1 day</option>
+											<option value="2 days">2 days</option>
+											<option value="3 days">3 days</option>
+											<option value="5 days">5 days</option>
+											<option value="1 week">1 week</option>
+											<option value="2 weeks">2 weeks</option>
+											<option value="1 months">1 month</option>
+											<! --<option value="permanent" class="font-weight-bold text-danger" selected>permanent</option>-->
+											<! --<option value="custom" class="font-weight-bold">custom</option> -->
```

Comment:
Breaking indent and commenting out _features_? Again, keep local changes to yourself.